### PR TITLE
Use correct infof calls in controller_manager

### PR DESCRIPTION
**Pull request by pmorie**
In a couple places we were incorrectly calling `Info` instead of `Infof`

### CODE REVIEW

File: charts/catalog/templates/apiregistration.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.useAggregator }}
-apiVersion: apiregistration.k8s.io/v1alpha1
+apiVersion: apiregistration.k8s.io/v1beta1
```
Comment (arschles) [inline charts/catalog/templates/apiregistration.yaml]: did this ever work on 1.6? I never tried it

**Comment by pmorie** [comment]
No, that should not be part of the change set :)


On Mon, Jun 19, 2017 at 6:22 PM, Aaron Schlesinger <notifications@github.com

File: charts/catalog/templates/apiregistration.yaml (hunk 1 above)
Comment (MHBauer) [inline charts/catalog/templates/apiregistration.yaml]: lawlz

**Comment by MHBauer** [comment]
changed my mind. we should use outfof. /badjoke

**Pull request merged**